desktop/release-notes.md

```diff
@@ -24,11 +24,85 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- Stop overriding .docker/config.json `credsStore` keys on application start. Note that if the developer uses a custom credential helper then the CLI `docker login` and `docker logout` will not affect whether the UI is signed in to Docker or not. In general it is better to sign into Docker via the UI since the UI supports multi-factor authentication.
+- Added a warning about the [forthcoming removal of Compose V1 from Docker Desktop](https://docs.docker.com/go/compose-v1-eol/). Can be suppressed with `COMPOSE_V1_EOL_SILENT=1`.
+- In Compose config, boolean fields in YAML should be either `true` or `false`. Deprecated YAML 1.1 values such as “on” or “no” will now produce a warning.
+- Improved UI for image table, allowing rows to use more available space.
+- Fixed various bugs in port-forwarding.
+- Fixed a HTTP proxy bug where an HTTP request without a Server Name Indication record would be rejected with an error.
+
+#### For Windows
+
+- Reverted to fully patching etc/hosts on windows (includes `host.docker.internal` and `gateway.docker.internal` again). For WSL, this behavior is controlled by a new General setting.
+  Fixes [docker/for-win#13388](https://github.com/docker/for-win/issues/13388) and [docker/for-win#13398](https://github.com/docker/for-win/issues/13398).
+- Fixed a spurious `courgette.log` file appearing on the Desktop when updating Docker Desktop. Fixes [docker/for-win#12468](https://github.com/docker/for-win/issues/12468).
+- Fixed "zoom in" shortcut (ctrl+=). Fixes [docker/for-win#13392](https://github.com/docker/for-win/issues/13392).
+- Fixed a bug where the tray menu would not correctly update after second container type switch. Fixes [docker/for-win#13379](https://github.com/docker/for-win/issues/13379).
+
+#### For Mac
+
+- Increase performance of VM networking when using Virtualization framework on macOS Ventura and above.
+- Fixed a bug where an error window is displayed on uninstall.
+- Fixed a bug where the setting `deprecatedCgroupv1` was ignored.
+- Fixed cases where `docker pull` would return `EOF`.
+
+#### For Linux
+
+- Fixed a bug where the VM networking crashes after 24h. Fixes [docker/desktop-linux#131](https://github.com/docker/desktop-linux/issues/131).
+
+### Security
+
+#### For all platforms
+
+- Fixed a security issue in Artifactory Integration where it would fallback to sending registry credentials over plain HTTP if HTTPS check failed. Fixes [docker/for-win#13344](https://github.com/docker/for-win/issues/13344).
```

Comment:
```suggestion
- Fixed a security issue in the Artifactory Integration where it would fallback to sending registry credentials over plain HTTP if HTTPS check failed. Fixes [docker/for-win#13344](https://github.com/docker/for-win/issues/13344).
```